nxdrive/manager.py

```diff
@@ -155,6 +155,7 @@ def __init__(self, home: Path, /) -> None:
             log.info(
                 ">>> Synchronization features are disabled, only Direct Edit and Direct Transfer will work."
             )
+        self.set_feature_state("synchronization", Options.synchronization_enabled)
```

Comment:
I think that line should be moved in the `if sync_enabled is not None` block. Because the default value of `Options.synchronization_enabled` is now `False`, and we need to think of users doing an upgrade and where the sync is enabled: it will no more be enabled then.